### ISSUE TITLE
[Feature] Add import/export of andOTP like json format (encrypted or unencrypted)

### ISSUE THREAD

**Issue by themipper** [2fast-team/2fast#12]
Love the windows app and use it in combination with andOTP on Android. Would love an import/export function for the format they are using. Would make creating backups and backup solutions so much faster :-)

**Comment by jp-weber** [comment]
I had already thought about this. but there are some problems. The system libraries for UWP do not support AES with mode GCM, so the decrypting the backup won't work. Another problem is the loading and saving case: If the import of the andOTP backup a onetime action, or will it synchronised?

I currently do not know how best to solve this without implementing a solution that is not satisfactory to me.

**Comment by p34c3-n-l0v3** [comment]
It would be really great to have import options for .json files from Aegis!

**Comment by ZeroDotNet** [comment]
I was about to open a feature request asking for the possibility of displaying the secret "code/key" for every account. 
Currently, I've to read every account QR with a barcode scanner app if I need to set up more than one OTP app. (Desktop, iOS, Laptop)

Perhaps this might be related to @themipper request? 
Let me know if it doesn't, and I'll enter a separate request.

**Comment by Daxx909** [comment]
Indeed, adding an option to show the secret code/key would make migrations to other apps possible, without having the complexity of different export formats.
Perhaps something like WinAuth had. It shows the code underneath the QR code, and you can copy it.
